web/blog/2023-02-21-junior-dev-misconceptions.md

```diff
@@ -0,0 +1,122 @@
+
+Nevertheless, in the "quality" debate there were effectively two camps, with those who thought quality code was about: 
+ 1. writing clean, readable code, that's easy to maintain 
+ 2. writing code that gets shipped on time and works.
+
+The balance between meeting deadlines, shipping features, and writing the best possible code is obviously a tricky one to get right. Some people had the opinion that business realities trump clean code patterns in the dash to meet deadlines and keep clients happy, while others thought that clean, quality code should be the priority, and that by making it a priority you can actually increase long-term velocity, even if short-term deadlines aren't met.
+
+<p align='center'>
+<figure>
+<img style={{ width: "100%" }} alt="Clean it up later" src="https://s3.us-west-2.amazonaws.com/secure.notion-static.com/62b1a947-d8ef-406a-9b33-a89b30e23af3/Untitled.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20230222%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20230222T194543Z&X-Amz-Expires=86400&X-Amz-Signature=ad7c767619af406e14e0183e0857eb7b8a75b94f2d07e95be7f998dd4ffed2d3&X-Amz-SignedHeaders=host&response-content-disposition=filename%3D%22Untitled.png%22&x-id=GetObject" />
+</figure>
+</p>
+
+This discussion can distract from Junior developers priorities though, which are to grow and improve as a developer, not lead the team to success. Therefore, it's probably best for Junior devs to focus on quality first, and then improve their speed of delivery second.
+
+## Staying Humble & Managing Expectations
+
+As a Junior developer, it's not expected that you're going to get everything right the first time. There is an assumption that you will learn the best practices over time, and along the way you might produce inconsistent work, make mistakes, or even possibly break some things along the way.
+
+But that's okay. 
+
+It's part of the process. It's expected. And it's important to remember that this is not a reflection of your value or worth as an engineer or individual. 
+
+In the replies, there were also many developers who recognized another developer's desire "to fix things later" as a way to brush off criticism towards their work. They generally viewed this as a bad habit to get into, as it is often one that plagues developers even as they gain more experience. "Code reviews are not personal", and being able to take criticism graciously is an important skill to develop. After all, seniors are there to guide you towards making better decisions based on their own experiences. And juniors are there to learn.
+
+<p align='center'>
```

Comment:
this img isn't showing when I run our blog locally?
![Screenshot 2023-02-23 at 11 32 09](https://user-images.githubusercontent.com/1536649/220882399-e4b17feb-26dd-42c4-b987-3edb8ba9d19d.png)